src/contour/CMakeLists.txt

```diff
@@ -1,11 +1,17 @@
 include(FilesystemResolver)
 
-find_package(Qt5 COMPONENTS Gui Network Widgets REQUIRED)  # apt install qtbase5-dev libqt5gui5
-
 option(CONTOUR_PERF_STATS "Enables debug printing some performance stats." OFF)
 option(CONTOUR_VT_METRICS "Enables collecting and exit-printing some VT usage metrics." OFF)
 option(CONTOUR_SCROLLBAR "Enables scrollbar in GUI frontend." ON)
 option(CONTOUR_BLUR_PLATFORM_KWIN "Enables support for blurring transparent background when using KWin (KDE window manager)." OFF)
+option(CONTOUR_BUILD_WITH_QT6 "Use Qt 6" OFF)
+
+if(CONTOUR_BUILD_WITH_QT6)
+   find_package(Qt6 COMPONENTS Core Gui Network OpenGL OpenGLWidgets Widgets REQUIRED)
+   set(CONTOUR_BLUR_PLATFORM_KWIN OFF)
```

Comment:
Ah, and why is this? What's the reason behind disabling?

Reply:
Can't compile with KF5. Maybe we should wait for KF6. :)